runtime/databricks/automl_runtime/forecast/model.py

```diff
@@ -57,10 +57,14 @@ def mlflow_forecast_log_model(forecast_model: ForecastModel,
     :param forecast_model: Forecast model wrapper
     :param sample_input: sample input Dataframes for model inference
     """
-    # log the model without signature if infer_signature is failed.
+    # TODO: we should not be logging without a signature since it cannot be registered to UC then
```

Comment:
Please add a JIRA ticket number to TODO:

e.g. # TODO(ML-XXXXX): we should not be logging...